src/test/java/Test2.java

```diff
@@ -0,0 +1,22 @@
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Test2 {
+    WebDriver driver = new ChromeDriver();
+
+    @Test
+    public void secondTest() {
+        driver.get("https://www.google.com.ua/search?q=w3 website");
```

Comment:
what page would this link leads you to? ' website' is extra here. (even though the page would still be correct)

Reply:
removed "website"

Reply:
changed